File: Java/q1018.java
// Cédulas

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int num = input.nextInt();
        
        System.out.println(num);
        int[] cedulas = {100, 50, 20, 10, 5, 2, 1};
        for (int i = 0; i < cedulas.length; i++) {
            int notas = num/cedulas[i];
            System.out.printf("%d nota(s) de R$ %d,00%n", notas, cedulas[i]);
            num = num - notas*cedulas[i];
        }
    }
}

File: Cpp/q1018.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int num, n;
    int notas[7] = {100, 50, 20, 10, 5, 2, 1};
    cin >> num;
    
    cout << num << endl;
    for(int nota : notas){
        n = (num/nota);
        cout << n << " nota(s) de R$ " << nota << ",00" << endl;
        num -= (n*nota);
    }
 
    return 0;
}
